tests/test_runs.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+from httpx import Response
+
+from prefect_census.credentials import CensusCredentials
+from prefect_census.runs import CensusGetSyncRunInfoFailed, get_census_sync_run_info
+
+
+@pytest.fixture
+def census_credentials():
+    return CensusCredentials(api_key="my_api_key")
+
+
+class TestGetCensusSyncRunInfo:
+    async def test_get_census_sync_run_info(self, respx_mock, census_credentials):
+        respx_mock.get(
+            "https://app.getcensus.com/api/v1/sync_runs/42",  # noqa
+            headers={"Authorization": "Bearer my_api_key"},
```

Comment:
This will need to be updated to accommodate Census' authentication method.

Reply:
Shoot, I forgot to post the links. I was looking at the [Census API docs ](https://docs.getcensus.com/basics/api#authenticating-api-requests) and the [Census implementation in the Prefect 1.0 task library](https://github.com/PrefectHQ/prefect/blob/e6bd04abb5580eb79bde12b9ad3845e9f1d1479c/src/prefect/tasks/census/census.py#L148) which led me to believe that the Census API token needed to be used in a basic auth format.  If this works, leave it this way, because this is a much better implementation than the version in the 1.0 task library.